share/wrf_timeseries.F

```diff
@@ -435,7 +459,8 @@ SUBROUTINE calc_ts( grid )
             grid%ts_u_profile(n,i,k)   = earth_u_profile(k)
             grid%ts_v_profile(n,i,k)   = earth_v_profile(k)
             grid%ts_w_profile(n,i,k)   = (grid%w_2(ix,k,iy)+grid%w_2(ix,k+1,iy))/2.0 ! w on cell center
```

Comment:
Wei,
This has W on full levels. Should that get modified like geopotential height?

Reply:
@davegill W is originally on full levels. Averaging k and k+1 values get it onto half levels.

Reply:
@weiwangncar 
Wei,
You are right - my mistake.